React-Projects/test-react-app/src/MyComponent.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from "react";
+
+export default function MyComponent() {
+  const [name, setName] = useState("N/A");
+  const [age, setAge] = useState(0);
+  const [comment, setComment] = useState("");
+  const [payment, setPayment] = useState("");
+  const [shipping, setShipping] = useState("Delivery");
+
+  function handleNameChange(event) {
```

Comment:
Use arrow function only if possible.

Reply:
@abhi-codal  Yes I forgot to use arrow function here. Will make sure from next time.